full/atm_land_ice_flux_exchange.F90

```diff
@@ -2390,6 +2523,13 @@ subroutine flux_down_from_atmos (Time, Atm, Land, Ice, Atmos_boundary, Land_boun
     call fms_xgrid_get_from_xgrid (Land_boundary%lprec,   'LND', ex_lprec,     xmap_sfc)
     call fms_xgrid_get_from_xgrid (Land_boundary%fprec,   'LND', ex_fprec,     xmap_sfc)
     call fms_xgrid_get_from_xgrid (Land_boundary%tprec,   'LND', ex_tprec,     xmap_sfc)
+
+    if (associated(Land_boundary%gex_fields)) then
+       do n_gex=1,n_gex_atm2lnd
+          call fms_xgrid_get_from_xgrid (Land_boundary%gex_fields(:,:,n_gex), 'LND', ex_gex_atm2lnd(:,n_gex), xmap_sfc)
+       end do
+    end if
```

Comment:
Shouldn't this be `Land_boundary%gex_atm2lnd` so that it can be consistent with L2493-L2502 above?